package/tests/conftest.py

```diff
@@ -87,6 +91,207 @@ def train_model(model_inputs, parameters):
     }
 
 
+@pytest.fixture
+def get_generic_pipe():
+    yield Pipeline(
+        [
+            node(
+                func=lambda x: x,
+                inputs="input_df",
+                outputs="output_df",
+                name="generic_node",
+            ),
+        ]
+    )
+
+
+# [TODO: Need to better manage this fixture]
```

Comment:
I see there's still a TODO here. The fixture is really large and quite hard to read IMO. Can we perhaps come up with example pipelines with more self-explanatory names and variable names? `dataset_1` etc is a bit hard to follow.. Or at least can we add more explicit notes that explain what each of these use_cases are for?

Reply:
Hi @merelcht , Yes even I felt the same. So for now kept a TODO and added links in the docstring of the fixture so the devs can take a look. I wanted to get some help from reviewers on how to do this more efficiently.

Based on your suggestion, I will try to come up with more meaningful names and see if we can split the fixture into multiple parts for each use case. Thank you